packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1262,10 +1262,13 @@ class Transform extends SingleChildRenderObjectWidget {
        alignment = null,
        super(key: key, child: child);
 
-  /// Creates a widget that scales its child uniformly.
+  /// Creates a widget that scales its child along the 2D plane.
   ///
-  /// The `scale` argument must not be null. It gives the scalar by which
-  /// to multiply the `x` and `y` axes.
+  /// The `scaleX` and `scaleY` provide the scalars to multiply the `x` and `y` axes.
+  ///
+  /// The `scale` provides the scalar to multiply both `x` and `y` axes and scales the child uniformly.
+  ///
+  /// Please note that, you can provide either only `scale` or both `scaleX` and `scaleY`, but all three parameters cannot be left null.
```

Comment:
You can remove "Please note that" per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Avoid "you" per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require

Reply:
I would say something like:

> Creates a widget that scales its child along the 2D plane.
>
> The `scaleX` argument provides the scalar by which to multiply the `x` axis, and the `scaleY` argument provides the scalar by which to multiply the `y` axis. Either may be omitted, in which case that axis defaults to 1.0.
>
> For convenience, instead of providing `scaleX` and `scaleY`, the `scale` parameter may be used.
>
> One of `scale`, `scaleX`, and `scaleY` must be non-null. If `scale` is provided, the other two must be null; similarly,  if it is not provided, one of the other two must be.

Reply:
Can I copy this whole thing? Cause it's like simply better.

Reply:
Can I copy this whole thing? This looks better.

Reply:
I commented in another review, but leaving the tail end as "must be" leaves the reader wondering whether you are referring to "must be provided" or "must be null". There are 2 extremely similar statements there and the antecedent could come either from the other part of the same phrase or the analogous part of the previous phrase - creating ambiguity.